Build/Projects/MonoGame.Framework.definition

```diff
@@ -44,6 +43,7 @@
     <CustomDefinitions>
       <Platform Name="Android">TRACE;ANDROID;GLES;OPENGL;OPENAL</Platform>
       <Platform Name="iOS">IOS;GLES;OPENGL;OPENAL</Platform>
+      <Platform Name="tvOS">IOS;__TVOS__;GLES;OPENGL;OPENAL</Platform>
```

Comment:
> To be fair the Xamarin Platform(s) already define these constants like this 

Sure... but if we went by what constants each platforms want to define for themselves they would all be inconsistent to each other.  IMO we either stick with our current convention or decide on a new one...  but we shouldn't just leave it to the whim of the platform we're on.

@KonajuGames ?